src/pages/503.astro

```diff
@@ -0,0 +1,39 @@
+---
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+import Layout from '../Layout.astro'
+import { Heading, Link, Paragraph } from '@amsterdam/design-system-react'
+---
+
+<Layout title="Er is iets mis gegaan">
+  <Heading level={1} className="ams-mb-l">Er is iets mis gegaan</Heading>
+  <Paragraph className="ams-mb-m" size="large">De pagina die u probeert te bezoeken heeft een storing.</Paragraph>
+  <Paragraph className="ams-mb--md">
```

Comment:
I spotted the margin bottoms looked a bit wrong, you have the wrong class name sometimes, I think because of an ADS update? Basically `ams-mb--md` should be `ams-mb-m` https://designsystem.amsterdam/?path=/docs/utilities-css-margin--docs

<img width="1644" alt="Screenshot 2025-04-15 at 09 38 42" src="https://github.com/user-attachments/assets/936d666d-dc0a-4543-910c-8434685fd472" />

Other than this looks good

Reply:
Thanks. I'll pick this up in a new branch as Niels is waiting for this one.

Reply:
Hi team! @thomasm0 kudo's for spotting such a detail! @SireeKoolenWijkstra this is such a tiny change to fix immediately, that in the future I expect such feedback to be addressed in the same PR. Otherwise, why ask for feedback? I'm sure @snelis was able to wait just a little bit longer.